<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlateSolveUI</class>
 <widget class="QDialog" name="PlateSolveUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plate Solving</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="0" colspan="3">
       <widget class="QPushButton" name="SolveButton">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Plate solve the image using the parameters below.</string>
        </property>
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="kcfg_FitsSolverUseScale">
        <property name="toolTip">
         <string>Set image scale to speed up solver as it does not have to search index files of different image scales.</string>
        </property>
        <property name="text">
         <string>Use Scale</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="kcfg_FitsSolverScale">
        <property name="toolTip">
         <string>Set image scale to speed up solver as it does not have to search index files of different image scales.</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="kcfg_FitsSolverImageScaleUnits">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;p&gt;The units of the imager scale bounds.</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>image width ยบ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>image width '</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>arcsec/pixel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="hSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>-1</width>
          <height>-1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="kcfg_FitsSolverUsePosition">
        <property name="toolTip">
         <string>Use the given position to speed up astrometry solver as it does not have to search in other areas of the sky.</string>
        </property>
        <property name="text">
         <string>Use Position</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="UpdatePosition">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set the approximate RA/DEC positions using the center position of the SkyMap.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="FitsSolverRALabel">
        <property name="toolTip">
         <string>The RA of the Estimated Telescope/Image Field Position in hh:mm:ss notation</string>
        </property>
        <property name="text">
         <string>RA</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="dmsBox" name="FitsSolverEstRA">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The RA of the Estimated Telescope/Image Field Position in hh:mm:ss notation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="hSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>-1</width>
          <height>-1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="FitsSolverDECLabel">
        <property name="toolTip">
         <string>The DEC of the Estimated Telescope/Image Field Position in dd:mm:ss notation</string>
        </property>
        <property name="text">
         <string>DEC</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="dmsBox" name="FitsSolverEstDec">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The DEC of the Estimated Telescope/Image Field Position in dd:mm:ss notation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <spacer name="hSpacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>-1</width>
          <height>-1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="FitsSolverRadiusLabel">
        <property name="toolTip">
         <string>The Search Radius for the Estimated Telescope/Image Field Position in degrees.</string>
        </property>
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="kcfg_FitsSolverRadius">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The Search Radius for the Estimated Telescope/Image Field Position in degrees.</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <spacer name="hSpacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>-1</width>
          <height>-1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="FitsSolverAngleLabel">
        <property name="toolTip">
         <string>The solved rotation angle in degrees.</string>
        </property>
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="FitsSolverAngle">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The solved rotation angle in degrees.</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <spacer name="hSpacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>-1</width>
          <height>-1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="FitsSolverProfileLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Selects the Options Profile (from Align) to use for Plate Solving</string>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QComboBox" name="kcfg_FitsSolverProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Selects the Options Profile (from Align) to use for Plate Solving</string>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <spacer name="hSpacer5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>-1</width>
          <height>-1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SolutionGroupBox">
     <property name="title">
      <string>Solution</string>
     </property>
     <layout class="QVBoxLayout" name="solutionLayout">
      <item>
       <widget class="QLabel" name="Solution1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Solution2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>-1</width>
       <height>-1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dmsBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/dmsbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
